ovirt_imageio/_internal/handlers/tickets.py

```diff
@@ -91,17 +91,24 @@ def delete(self, req, resp, ticket_id):
         requests in case of network failures. See
         https://tools.ietf.org/html/rfc7231#section-4.2.2.
         """
-        log.info("[%s] REMOVE ticket=%s", req.client_addr, ticket_id)
-
         if ticket_id:
             try:
-                self.auth.remove(ticket_id)
-            except errors.TicketCancelTimeout as e:
-                # The ticket is still used by some connection so we cannot
-                # remove it. The caller can retry the call again when the
-                # number connections reach zero.
-                raise http.Error(http.CONFLICT, str(e))
+                ticket = self.auth.get(ticket_id)
+            except KeyError:
+                # Ticket was already deleted.
+                log.info("[%s] REMOVE transfer=(deleted)", req.client_addr)
+            else:
+                log.info("[%s] REMOVE transfer=%s",
+                         req.client_addr, ticket.transfer_id)
+                try:
+                    self.auth.remove(ticket_id)
+                except errors.TicketCancelTimeout as e:
+                    # The ticket is still used by some connection so we cannot
+                    # remove it. The caller can retry the call again when the
+                    # number connections reach zero.
```

Comment:
number connections
-->
number of connections

Reply:
Yes, but this is not related to the change. However errors.TicketCancelTimeout was renamed to errors.TransferCancelTimeout, so we may have a bug here.